packages/client-ts/src/context/engines/verida/messaging/engine.ts

```diff
@@ -74,7 +74,7 @@ class MessagingEngineVerida implements BaseMessage {
     // Ping the notification service if it exists
     // @todo: Make it configurable if the notification service is pinged
     if (response && this.notificationService) {
-      await this.notificationService.ping()
+      await this.notificationService.ping(config, did);
```

Comment:
Can we pass just the context name from the config instead of the whole config object? That way we minimise the interdependency between the different classes.

Reply:
@tahpot  The implemented `ping(config?:MessageSendConfig, ...)` Current clients reads the config object. This will limit the configuration we can pass here.